nabl2.lang/build.gradle.kts

```diff
@@ -3,6 +3,6 @@ plugins {
   `maven-publish`
 }
 
-spoofax {
-  createPublication = true
+spoofaxLanguageSpecification {
```

Comment:
`createPublication` defaults to `true` now, so the entire `spoofaxLanguageSpecification` blocks can just be removed.